Jenkinsfile

```diff
@@ -43,6 +45,18 @@ timestamps {
     } // stage
 
     stage('Setup') {
+
+      // download NDK
```

Comment:
uh, so this block makes no sense whatsoever here.You're requesting a linux node and an OS X node to grab the ndk and unzip it, but not keeping the result. 

Basically this says: Give me a 'linux' (or 'osx') machine, create a temporary workspace/directory, download the idk and unzip it (and then clean up the workspace and free up the machine)